src/derivatives/05-diffusion-derivatives.md

```diff
@@ -41,83 +41,81 @@ which has its own unique requirements with respect to:
-    *model parameters*, without necessitating reference to
-    the empirical diffusion-weighted data.
+- Files "`<source_keywords>[_space-<space>]_model-<label>_param-<param*>_dwimap.nii[.gz]`"
+    provide model outputs and/or parameters of the model.
 
-For example, consider a diffusion tensor model fit: the number of
-iterations in the optimization algorithm would be an *input* parameter;
-the six unique diffusion tensor coefficients would be the *model fit*
-parameters; the Fractional Anisotropy (FA) would be a *model-derived*
-parameter (as it is calculated from the diffusion tensor coefficients
-rather than the image data).
+- Only one metadata file, "`<source_keywords>[_space-<space>]_model-<label>_dwimap.json`"
+    is allowed for each set of files that has a specific `model-<label>` entity.
 
-### File names
+Example:
 
 ```Text
-<pipeline_name>/
-    sub-<participant_label>/
+my_tensors/
+    sub-01/
         dwi/
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_model.json
-
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.json]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.json]
-```
+            sub-01_model-DTI_param_Sxx_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxy_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxz_dwimap.nii.gz
+            sub-01_model-DTI_param_Syy_dwimap.nii.gz
+            sub-01_model-DTI_param_Syz_dwimap.nii.gz
+            sub-01_model-DTI_param_Szz_dwimap.nii.gz
+            sub-01_model-DTI_dwimap.json
+            sub-01_model-DTI_param-FA_dwimap.nii.gz
+            sub-01_model-DTI_param-MD_dwimap.nii.gz
 
--   Files "`<source_keywords>[_space-<space>]_model-<label>_param-<param*>_model.nii[.gz]`"
-    provide data corresponding to the various requisite [model fit parameters](#paramdef-fit).
+```
 
-    In cases where *all* [model fit parameters](#paramdef-fit) are contained within a single image
-    file, entity "`_param-`" MUST NOT be included; e.g.:
-        ```Text
-        <pipeline_name>/
-            sub-<participant_label>/
-                dwi/
-                    <source_keywords>[_space-<space>]_model-<label>_model.nii[.gz]
-                    <source_keywords>[_space-<space>]_model-<label>_model.json
-        ```
+In cases where a set of different parameters are contained within a single image
+file, entity "`_param-`" MUST NOT be included; e.g.:
```

Comment:
I think this may be an erroneous translation of the purpose of the text being replaced into the newer paradigm.

The original text can be re-phrased as follows. The purpose of the `_param-` entity (as with basically all entities in BIDS) is *disambiguation*: to separate different parameters that are a part of the same model. In the scenario where all model fit parameters are encoded into a single data file, such disambiguation is no longer necessary, and so it makes reasonable sense to omit it.

The problem here then is two-fold:

1.  As a nuanced criticism of the current phrasing, it's not specifically that "multiple parameters" are encoded in a single data file; it's specifically that "*all* parameters" are encoded in a single data file.
    There may be cases where a single data file may indeed "contain" multiple parameters. I tried to transition the proposal away from such as much as possible, since if different parameters have different interpretations in terms of units or how they encode anisotropy it's more sensible to split them across data files. But maybe there's some sensible use case where a data file may contain multiple parameters but not be the only data file. In that hypothetical case I'd argue that omitting the `param` entity for that data file doesn't really communicate what's going on. Indeed it sort of implicitly provides the argument against such internal combination of distinct parameters.

2.  There were reasonable scenarios where this condition would previously have applied, since it's possible that for some models (eg. tensor, single-tissue CSD) the entirety of the model fit may be encoded in a single image file. However where `dwimap` provides no distinction between model fit and model-derived, for this condition to apply it would be necessary *both* for this to be the case *and* for there to be no other parametric maps yielded from that model (ie. formerly model-derived parameters). So it might be simpler overall if the `param` entity is simply compulsory.
    As proof that this doesn't oblige nonsensical entity values in trivial cases:
    1.  A tensor image encodes diffusivity; it just uses a tensor representation (according to #74).
    2.  A single-tissue ODF encodes AFD; it just uses a SH representation.

Reply:
I agree. How about `param-tensor` for the six tensor components and `param-sh` for the spherical harmonic coefficients?

Reply:
`param-tensor` and `param-sh` is the opposite of what I'm advocating. Those are data representations describing how anisotropic information on S2 may be encoded. They don't say anything about the quantitative parameter being encoded. Those are the diffusion coefficient and AFD respectively.

(This might seem pedantic for these easier cases, but I've spent a lot of time thinking about the consequences of these decisions for more complex models)